File: Discord-QR-Token-Logger-master/Discord-QR-Token-Stealer/infoPanel.cs
using System;
using System.Windows.Forms;
using System.Diagnostics;

namespace Discord_QR_Token_Stealer
{
    public partial class infoPanel : Form
    {
        public infoPanel()
        {
            InitializeComponent();
        }

        void openUrl(string url)
        {
            var proc = new Process();
            var info = new ProcessStartInfo();
            info.FileName = url;
            proc.StartInfo = info;
            proc.Start();
        }

        private void label1_Click_1(object sender, EventArgs e)
        {
            openUrl("https://verlox.cc");

        }

        private void label2_Click(object sender, EventArgs e)
        {
            openUrl("https://github.com/verlox");
        }

        private void linkLabel1_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
        {
            Process.Start("https://www.youtube.com/channel/UCRoxxUj5BCM4DPP9aF7jQ9w");
        }

        private void linkLabel2_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
        {
            Process.Start("https://www.facebook.com/developers.syriaa");
        }

        private void linkLabel4_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
        {
            Process.Start("https://t.me/MONSTERMCSY");
        }

        private void linkLabel3_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
        {
            Process.Start("https://magholarabeee.blogspot.com");
        }
    }
}

File: SRC VB/Vb/Discord-QR-Token-Stealer/infoPanel.vb
Imports System
Imports System.Windows.Forms
Imports System.Diagnostics

Namespace Discord_QR_Token_Stealer
	Partial Public Class infoPanel
		Inherits Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub openUrl(ByVal url As String)
			Dim proc = New Process()
			Dim info = New ProcessStartInfo()
			info.FileName = url
			proc.StartInfo = info
			proc.Start()
		End Sub

		Private Sub label1_Click_1(ByVal sender As Object, ByVal e As EventArgs) Handles label1.Click
			openUrl("https://verlox.cc")

		End Sub

		Private Sub label2_Click(ByVal sender As Object, ByVal e As EventArgs) Handles label2.Click
			openUrl("https://github.com/verlox")
		End Sub
	End Class
End Namespace
